CardForm/src/test/java/com/braintreepayments/cardform/view/CvvEditTextTest.java

```diff
@@ -68,6 +70,28 @@ public void hintChangesForCardType() {
         }
     }
 
+    @Test
+    public void appliesPasswordInputTypeWhenMasked() {
```

Comment:
Input type might be a little confusing here since it's actually a transformation and a password input type is slightly different.

Reply:
We initially used input type, which is why the test is named that way. I'm going to revert cvv to use input type, so I'll keep this test the same.